docker/.dockerignore

```diff
@@ -0,0 +1 @@
+histfile
```

Comment:
what is `histfile`?

Reply:
bash history as set by docker-compose.yml